doc/source/user_guide/configuration.rst

```diff
@@ -66,4 +68,21 @@ class:
 .. autopydantic_model:: Credentials
   :model-show-config-summary: False
   :model-show-validator-summary: False
+  :model-show-validator-members: False
   :model-show-json: False
+  :field-list-validators: False
+
+.. _Interactive mode:
+
+Interactive mode
+------------------
+
+When the property `interactive` is set to `true`, the users could enter configuration
+properties from the terminal as they get prompted.
```

Comment:
```suggestion
When the property `interactive` is set to `true`, the user will be prompted for missing configuration options.
```

Reply:
I updated this part slightly different from this suggestion but along the same lines.